### PR TITLE
mapfishapp - gml+kml export of results

### PR DESCRIPTION
**Pull request by fvanderbiest**
This PR allows users to export features from the results panel as KML or GML files.

### CODE REVIEW

File: mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java [hunk 1]
```diff
@@ -142,6 +152,27 @@ public void getKMLFile(HttpServletRequest request, HttpServletResponse response)
...
 
+    /*======================= GML =====================================================================*/
+    /**
+     * POST KML entry point. Store the body of the request POST (or file by upload) in a temporary file.
+     * @param request contains in its body the file in the JSON format
+     * @param response contains the url path to get back the file in CSV: GML_URL/{filename}
+     */
+    @RequestMapping(value="/gml/", method=RequestMethod.POST)
+    public void storeGMLFile(HttpServletRequest request, HttpServletResponse response) {
+        storeFile(new GMLDocService(this.docTempDir, this.connectionPool), GML_URL, request, response);
```
Comment (pmauduit) [inline mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java]: Missing class GMLDocService

Comment (fvanderbiest) [inline mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java]: True, should now be fixed.

File: mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java [hunk 2]
```diff
@@ -142,6 +152,27 @@ public void getKMLFile(HttpServletRequest request, HttpServletResponse response)
         getFile(new KMLDocService(this.docTempDir, this.connectionPool), request, response);
     }
 
+    /*======================= GML =====================================================================*/
+    /**
+     * POST KML entry point. Store the body of the request POST (or file by upload) in a temporary file.
```
Comment (pmauduit) [inline mapfishapp/src/main/java/org/georchestra/mapfishapp/ws/DocController.java]: s/kml/gml/ ;-)

**Comment by fvanderbiest** [comment]
FIXME:
 * check export against raster layers
 * missing imports
 * missing i18n

**Pull request merged**